en/core-libraries/caching.rst

```diff
@@ -478,7 +478,7 @@ as in plugins using ``$plugin\Cache\Engine``. src/plugin cache engines can
 also override the core engines. Cache adapters must be in a cache
 directory. If you had a cache engine named ``MyCustomCacheEngine``
 it would be placed in either **src/Cache/Engine/MyCustomCacheEngine.php**
-as an app/libs. Or in **plugin/Cache/Engine/MyCustomCacheEngine.php** as
+as an app/libs. Or in **plugin/src/Cache/Engine/MyCustomCacheEngine.php** as
```

Comment:
Shouldn't it ``plugins/CachePack/src/Cache/Engine/MyCustomCacheEngine.php``?

Reply:
Yes it should. As there is written "plugin" i asumed its meant to be something like "MyPlugin" and not representing the plugins folder.